gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -39,6 +39,7 @@ from google.api_core import grpc_helpers_async
 from google.api_core import path_template
 {% if service.has_lro %}
 from google.api_core import future
+from google.api_core import operation
```

Comment:
Probably a stupid question, but whasn't this required before? I.e. it is logical to assume that `{% if service.has_lro %}` that operation should be imported.

Reply:
It's a different operation module :P